stellar/operations.py

```diff
@@ -36,14 +36,9 @@ def copy_database(raw_conn, from_database, to_database):
     terminate_database_connections(raw_conn, from_database)
 
     if raw_conn.engine.dialect.name == 'postgresql':
-        raw_conn.execute(
-            '''
-                CREATE DATABASE "%s" WITH TEMPLATE "%s";
-            ''' %
-            (
-                to_database,
-                from_database
-            )
+        sqlalchemy_utils.functions.create_database(
+            '%s%s' % (raw_conn.engine.url, to_database),
+            template=from_database
```

Comment:
The point was not about relying on something that is already listed in the dependency list but about being more flexible on versions sqlalchemy-utils required.
After three years I think this point is no longer valid (and I even wonder how/why you digged up this comment ^^)